src/editor/EditorCommandHandlers.js

```diff
@@ -237,6 +241,27 @@ define(function (require, exports, module) {
         return {edit: editGroup, selection: trackedSels};
     }
     
+    
+    /**
+     * @private
+     * Given a line to be commented it will determine where the comment prefix position is.
+     * 
+     * @param {string} language id
+     * @param {string} line to be commented
+     * @return {integer}
+     */
+    function _getPrefixPos(language, line) {                        
+        if (language === "jade") {
```

Comment:
EditorCommandHandlers shouldn't have hardcoded knowledge of specific languages -- this should be a general flag, e.g. `Language.indentLineComments`.  I'd imagine this same issue affects braceless/classic SASS too, for example.